ajax/libs/skeletonic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "skeletonic",
+  "version": "1.0.89",
+  "description": "Skeletonic is a lightweight, intuitive and ultra-responsive CSS framework to speed up your Digital and Mobile Web development needs.",
+  "filename": "skeletonic.min-1.0.89.css",
+  "author": "Reedia Limited",
+  "homepage": "https://skeletonic.io",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/reedia/skeletonic"
+  },
+  "keywords": [
+    "skeletonic",
+    "skeletonic.min-1.0.89.css",
+    "CSS",
+    "lightweight",
+    "dead on",
+    "responsive",
+    "framework",
+    "semantic",
+    "mobile-first",
+    "modern",
+    "style-agnostic",
+    "front-end",
+    "frontend",
+    "grid system",
+    "typography"
+  ],
+  "license": "MIT",
+  "npmName": "skeletonic",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.css"
```

Comment:
Not sure do users need fonts in some old versions?
Because I saw https://github.com/reedia/skeletonic/blob/v1.0.69/dist/skeletonic-1.0.67.css#L333
If they need, let's use `**/!(*.html)` to only exclude html file.
If they don't, let's use `!(fonts|*.html){,**/*}` to exclude html file and fonts.

P.S. We usually use exclusive pattern for the distribution folder like `dist` because we assume author(s) only put files for users under it.

Reply:
@extend1994 

Thanks for your thorough feedback, very much appreciated! 😄 

1) Version number has been removed from the filename which to be honest makes perfect sense and also simplify integration. This is applied to version v1.0.93 at https://github.com/reedia/skeletonic/releases/tag/v1.0.93

2) I have also removed the index.html on my GitHub and would appreciate if you could help removing it as well.

3) Fonts are not required just provided as an example on how to use the library so happy with adding ```!(fonts|*.html){,**/*}```

Many thanks again